.azure/scale.csv

```diff
@@ -0,0 +1,26 @@
+stage,siteId,aksArc-lnet-dnsServers,phone,domainServerIP,streetAddress1,enableProvisioners,aksArc-lnet-addressPrefix,adouSuffix,endingAddress,dcPort,defaultGateway,managementAdapters,rdmaEnabled,virtualHostIp,aksArc-lnet-endingAddress,startingAddress,storageConnectivitySwitchless,companyName,location,serverPorts,aksArc-controlPlaneIp,mobile,servers,postalCode,contactName,domainFqdn,emailList,city,dnsServers,aksArc-lnet-defaultGateway,country,storageNetworks,aksArc-lnet-startingAddress,authenticationMethod,rbacAdminGroupObjectIds,stateOrProvince
```

Comment:
## Arc IaC Automation generated
Next steps: 
  - [ ] Setup local git hook by running command in the root folder of cloned repo: git config --local core.hooksPath ./.azure/hooks/  
  - [ ] Fill the csv file and commit, the struct of the csv file can be found on [here](https://github.com/Azure/Edge-infrastructure-quickstart-template/blob/main/doc/Scale-with-automation.md#methodology)
  - [ ] Wait for the scale workflow run, and then resolve this comment and merge the pull request

 If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com